releaselinks.go

```diff
@@ -0,0 +1,167 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/url"
+)
+
+// ReleaseLinksService handles communication with the release link methods
+// of the GitLab API.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html
+type ReleaseLinksService struct {
+	client *Client
+}
+
+// ReleaseLink represents a release link.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html
+type ReleaseLink struct {
+	ID       int    `json:"id"`
+	Name     string `json:"name"`
+	URL      string `json:"url"`
+	External bool   `json:"external"`
+}
+
+// ListReleaseLinksOptions represents ListReleaseLinks() options.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html#get-links
+type ListReleaseLinksOptions ListOptions
+
+// ListReleaseLinks gets assets as links from a Release.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html#get-links
+func (s *ReleaseLinksService) ListReleaseLinks(pid interface{}, tag string, opt *ListReleaseLinksOptions, options ...OptionFunc) ([]*ReleaseLink, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/releases/%s/assets/links", url.QueryEscape(project), tag)
+
+	req, err := s.client.NewRequest("GET", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var rls []*ReleaseLink
+	resp, err := s.client.Do(req, &rls)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return rls, resp, err
+}
+
+// GetReleaseLink returns a link from release assets.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html#get-a-link
+func (s *ReleaseLinksService) GetReleaseLink(pid interface{}, tag string, linkID int, options ...OptionFunc) (*ReleaseLink, *Response, error) {
```

Comment:
Following the conventions used in this package `tag` should be `tagName` and `linkID`, should just be `link`. This goes for all occurrences of course.